renovate.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "reviewers": [
+    "team:gotify"
+  ],
+  "enabledManagers": [
+    "gomod",
+    "dockerfile",
+    "custom.regex"
+  ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^GO_VERSION$"
+      ],
+      "depTypeTemplate": "language",
+      "matchStrings": [
+        "^(?<currentValue>.+)\\n"
+      ],
+      "depNameTemplate": "https://github.com/golang/go",
```

Comment:
It would be good to automate the gotify/build image creation, as otherwise we break the build when we merge an PR updating the GO_VERSION file.